src/Display/Modal/Modal.test.tsx

```diff
@@ -31,14 +31,19 @@ const ModalComponent = ({ isVisible }: { isVisible: boolean }) => (
   </Modal>
 );
 
-function setupModal(isVisible: boolean) {
+type TestProps = Omit<Props, 'onClose'> & {
+  onClose?: any;
```

Comment:
The type has to be any?

Reply:
Oh just realized it could be `onClose?: () => void`.

Previously, it is declared as `onClose()?: void` which will make TS complain the line of `onClose: null` in the test file.